.github/workflows/test.yml

```diff
@@ -89,8 +89,8 @@ jobs:
 
       - name: Install SpacetimeDB CLI from the local checkout
         run: |
-          cargo install --path SpacetimeDB~/crates/cli --locked --message-format=short
-          cargo install --path SpacetimeDB~/crates/standalone --locked --message-format=short
+          cargo install --force --path SpacetimeDB~/crates/cli --locked --message-format=short
+          cargo install --force --path SpacetimeDB~/crates/standalone --locked --message-format=short
```

Comment:
I don't know why but this was required to get it to stop complaining about `spacetime` already existing

Reply:
I'm not overly concerned.